app/assets/stylesheets/components/footer.scss

```diff
@@ -0,0 +1,149 @@
+@import "variables_mixins";
+
+.footer {
+	background-color: $base-text;
```

Comment:
indents have gone mega big, they are 2 spaces elsewhere - if theres a way we can enforce this that'd be great (prettier?)